travis-ci/install.sh

```diff
@@ -20,6 +20,8 @@ cd $DRUPAL_BUILD_ROOT/drupal
 
 # Set our drupal core version.
 composer require drupal/core $DRUPAL_CORE --no-update
+if [[ $DRUPAL_CORE = '8.4.x-dev' ]]; then composer --verbose remove --no-update drupal/console; fi;
+if [[ $DRUPAL_CORE = '8.4.x-dev' ]]; then composer --verbose require --no-update drush/drush:9.0.x-dev; fi;
```

Comment:
We don't need console or drush for any tests, so let's ALWAYS remove them (including a comment as to why)